main.tf

```diff
@@ -161,3 +161,47 @@ resource "google_monitoring_uptime_check_config" "global-uptime-check" {
     create_before_destroy = true
   }
 }
+
+// Create an alert policy based on the uptime check above.
+resource "google_monitoring_alert_policy" "prober_uptime" {
+  project = var.project_id
+  # In the absence of data, incident will auto-close in 7 days
+  alert_strategy {
```

Comment:
I was intentionally leaving this out of scope for this module.  Perhaps in the example we can show how to wrap this up with monitoring instead?  I figured we'd add this in our downstream wrapper module.

Reply:
got it, thanks for the clarification, will close this PR and add the alert policy as example